### PR TITLE
Bump psutil and sagemaker-containers version

### PR DESCRIPTION
**Pull request by rizwangilani**
Bump psutil from 5.4.8 to 5.6.7 and sagemaker-containers from 2.5.6 to 2.8.3

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,11 +3,11 @@ gunicorn<20.0.0
 matplotlib
 numpy
 pandas>=0.24.0
-psutil==5.4.8  # sagemaker-containers requires psutil 5.4.8
+psutil==5.6.6  # sagemaker-containers requires psutil 5.4.8
```
Comment (edwardjkim) [inline requirements.txt]: Is the comment `sagemaker-containers requires psutil 5.4.8` still true?

Comment (rizwangilani) [reply requirements.txt]: actually, sagemaker-containers is using psutil `5.6.7` in their` 2.8.3` version. updated PR.

**Comment by edwardjkim** [comment]
- Did we run integration tests?
- Does sagemaker 2.8.3 have the fix for the requirements.txt issue? Is there a way for us to test this?
- Does XGBoost have network isolation?

**Comment by aws-patlin** [comment]
@edwardjkim Just an FYI, the issues with requirements.txt and network isolation should not affect XGBoost. Those bugs were present in the code path of `modules.download_and_install` and `entrypoint.run`, while in XGBoost we call `modules.run_module`.

**Comment by edwardjkim** [comment]
Thanks @aws-patlin, that makes sense. Can we just make sure to manually run integration tests as a sanity check before merging since this involves sagemaker-containers?

**Pull request merged**